format/spec.md

```diff
@@ -128,13 +128,13 @@ Tables do not require rename, except for tables that use atomic rename to implem
 
 #### Writer requirements
 
-Some tables in this spec have columns that specify requirements for v1 and v2 tables. These requirements are intended for writers when adding metadata files to a table with the given version.
+Some tables in this spec have columns that specify requirements for v1 and v2 tables. These requirements are intended for writers when adding metadata/manifest files to a table with the given version.
 
-| Requirement | Write behavior |
-|-------------|----------------|
-| (blank)     | The field should be omitted |
-| _optional_  | The field can be written |
-| _required_  | The field must be written |
+| Requirement | Write behavior                                        |
+|-------------|-------------------------------------------------------|
+| (blank)     | The field should not be present in the schema         |
```

Comment:
I think we cannot mention schema? Because Table metadata is a JSON document and we no where keep JSON schema of that document. For Avro it makes sense. But this section is generic I guess.

Reply:
I would argue that the structure of the JSON document itself represents the schema, but it isn't separated from the data itself.

Reply:
See, it's already two people seeing it differently ;). I guess that warrants that we are more precise here, so that such misunderstandings don't arise when reading the spec.

I think it is absolutely okay to be concrete w.r.t. the data formats in this section, especially since JSON is not a valid Iceberg file format, therefore we also cannot just use Iceberg terminology without defining how exactly it applies for JSON.

Reply:
Okay, I've removed schema to avoid confusion.